tests/DotNet.Testcontainers.Tests/Assets/credHelpers/docker-credential-password.bat

```diff
@@ -0,0 +1,2 @@
+@echo off
+echo {"Username":"username","Secret":"password"}
```

Comment:
I really like the idea with the batch and bash scripts. I think we can use the same approach to test 
the `CredsStoreProvider` implementation ✌️. We can move the part that prepares the environment in a class fixture or something like that (in the future). That makes it a bit cleaner. Great idea.